core/src/test/java/org/apache/stormcrawler/bolt/FeedParserBoltTest.java

```diff
@@ -51,7 +51,10 @@ void testFeedParsing() throws IOException {
         Metadata metadata = new Metadata();
         // specify that it is a Feed file
         metadata.setValue(FeedParserBolt.isFeedKey, "true");
-        parse("http://www.guardian.com/Feed.xml", "guardian.rss", metadata);
+        parse(
+                "https://de.wikipedia.org/w/index.php?title=Apache_Software_Foundation&action=history&feed=rss",
```

Comment:
can we use some page listed under apache scope instead? not sure if using wikipedia might upset some as well

Reply:
Do you have an example for an RSS feed on an apache site ?

Reply:
Alternative is I will just create something random myself.

Reply:
Let me create one for stormcrawler.apache.org (just to be save)

Reply:
maybe comdev has some
https://community.apache.org/blog/

the latest says
https://community.apache.org/blog/rss-feeds-activated.html

Reply:
Thanks. Think we are good now. I replaced it with handmade rss based upon the sitemap of our actual stormcrawler website.